flexget/plugins/sites/lostfilm.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+import re
+
+import feedparser
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.utils.requests import RequestException
+from flexget.utils.soup import get_soup
+
+__author__ = 'danfocus'
+
+log = logging.getLogger('lostfilm')
+
+EPISODE_REGEXP = re.compile(
+    '.*lostfilm.tv/series/.*/season_(\d+)/episode_(\d+)/.*')
+LOSTFILM_ID_REGEXP = re.compile(
+    '.*static.lostfilm.tv/Images/(\d+)/Posters/poster.jpg.*')
+TEXT_REGEXP = re.compile('^(\d+)\sсезон\s(\d+)\sсерия\.\s(.+)\s\((.+)\)$')
+
+quality_map = {
+    'SD': ('480p', 'avi'),
+    '1080': ('1080p', 'mkv'),
+    'MP4': ('720p', 'mp4'),
+    'HD': ('720p', 'mkv')
+}
+
+LOSTFILM_URL = 'http://lostfilm.tv/rss.xml'
+
+
+class LostFilm(object):
+    """
+    Change new lostfilm's rss links
+
+    Example::
+
+      lostfilm: yes
+
+    Advanced usage:
+
+      lostfilm:
+        url: <url>
+    """
+
+    schema = {
+        'type': ['boolean', 'object'],
+        'properties': {
+            'url': {'type': 'string', 'format': 'url'},
+        },
+        'additionalProperties': False
+    }
+
+    def build_config(self, config):
+        """Set default url to config"""
+        if isinstance(config, bool) and config is True:
+            config = {'url': LOSTFILM_URL}
+        return config
+
+    def on_task_input(self, task, config):
+        config = self.build_config(config)
+        if config is False:
+            return
+        rss = feedparser.parse(config['url'])
+        if rss.get('status') != 200:
+            log.error('Received not 200 (OK) status')
+            return
+        entries = []
+        for item in rss.entries:
+            if item.get('link') is None:
+                log.debug('Item doesn\'t have a link')
+                continue
+            if item.get('description') is None:
+                log.debug('Item doesn\'t have a description')
+                continue
+            match = LOSTFILM_ID_REGEXP.search(item['description'])
+            if match:
+                lostfilm_num = match.group(1)
+            else:
+                log.debug('Item doesn\'t have lostfilm id in description')
+                continue
+            match = EPISODE_REGEXP.match(item['link'])
+            if match:
+                season_num = int(match.group(1))
+                episode_num = int(match.group(2))
+            else:
+                log.debug('Item doesn\'t have episode id in link')
+                continue
+            redirect_url = 'http://www.lostfilm.tv/v_search.php?' \
+                'c={}&s={}&e={}'.format(lostfilm_num, season_num, episode_num)
```

Comment:
probably best to pass this as a dict param:
```python
params = {
    'c': lostfilm_num,
    's': season_num,
    'e': episode_num
}
```
then add it to request with `query` keyword:
```python
response = task.requests.get(redirect_url, query=params)
```